free-1.md

```diff
@@ -56,13 +155,33 @@ spent directly engaging in the **language** aspects of this project (e.g.,
 making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
+I am not entirely sure how to implement languages quite yet, so I don't think I
+have an accurate judgment of this at the moment. I do, however, think that this
+may require a lot of systems code to work in the context of an editor. At this
```

Comment:
I suspect it will end up taking relatively little language design; your language is basically JSON.